src/sentry.tsx

```diff
@@ -0,0 +1,36 @@
+import * as Sentry from "@sentry/browser";
+import { Integrations as ApmIntegrations, Span } from "@sentry/apm";
+import { CaptureConsole as CaptureConsoleIntegration } from "@sentry/integrations";
+
+const Tracing = ApmIntegrations.Tracing;
+
+export const setupSentry = (options: { [key: string]: any }) => {
+  console.log(`[sentry] Initialized with DSN: ${options.dsn}`);
+  Sentry.init({
+    tracesSampleRate: 1.0,
+    ...options,
+    integrations: [
+      new CaptureConsoleIntegration({
+        levels: ["error"],
+      }),
+      new ApmIntegrations.Tracing(),
+    ],
+  });
+};
+
+export const startSpan = (options: {
+  description?: string;
+  op?: string;
+}): Span | null => {
+  const tracingIntegration = Sentry.getCurrentHub().getIntegration(Tracing);
+  if (!tracingIntegration) {
+    console.warn("startSpan called without tracing integration");
+    return null;
+  }
+  const transaction = (tracingIntegration as any).constructor.getTransaction();
+  if (!transaction) {
+    console.warn("startSpan called without transaction");
```

Comment:
doesnt need to be a warning as its super common..

i also likely need to add new styles of transactions here. not sure what yet, but maybe when someone presses a tile its a new transaction. will experiment and see what can create value for patterns.